acr-build.yml

```diff
@@ -0,0 +1,28 @@
+version: v1.1.0
+steps:
+  - id: ccd
+    build: >
+      -t {{.Run.Registry}}/ccd/logstash/ccd
+      --target ccd
+      --build-arg conf=ccd_logstash.conf.in
+      .
+    when:
+      - pull-ccd
```

Comment:
@timja as I've removed the pull step should this now be `["-"]` or is there a different value?

Reply:
just remove the when, see the simpler example I sent you https://github.com/hmcts/cnp-jenkins-docker/blob/master/jenkins-agent/acr-agents-build-task.yaml